apps/website/src/pages/events/fall-carnival-2023/tickets/[userName].tsx

```diff
@@ -0,0 +1,158 @@
+
+  const ticket = trpc.virtualTickets.getTicket.useQuery(
+    { eventId, userName: userName as string },
+    { enabled: !!userName, retry: 2 },
+  );
+
+  const showTicket = userName && ticket.data;
+
+  const ogImage =
+    env.NEXT_PUBLIC_BASE_URL +
+    getVirtualTicketImageUrl(eventId, userName, "og");
+
+  return (
+    <>
+      <Meta
+        title="Alveus Fall Carnival 2023"
+        description="You are invited to the Alveus Fall Carnival 2023 - Customize your ticket!"
+      >
+        {/* preload the ticket */}
+        <link
+          rel="preload"
+          as="image"
+          href={
+            env.NEXT_PUBLIC_BASE_URL +
+            getVirtualTicketImageUrl(eventId, userName)
+          }
+        />
+
+        {ogImage && (
+          <meta key="og:image" property="og:image" content={ogImage} />
+        )}
+        {ogImage && (
+          <meta key="twitter:image" name="twitter:image" content={ogImage} />
+        )}
+      </Meta>
+
+      {/* Nav background */}
+      <div className="-mt-40 hidden h-40 bg-[#4E1362] bg-gradient-to-t from-[#4E1362] to-[#390E47] lg:block" />
+
+      <div className="relative">
+        <Image
+          src={leafLeftImage2}
+          alt=""
+          className="pointer-events-none absolute -bottom-24 left-0 z-10 hidden h-auto w-1/2 max-w-[12rem] select-none lg:block"
+        />
+
+        <IntroSection showLinkToClaimTicket={true} />
+      </div>
+
+      <Section containerClassName="flex flex-col items-center text-center">
+        {ticket.error && <p>Could not find ticket!</p>}
+        {!ticket.error && (
+          <>
+            <p className="max-w-xl text-lg md:text-xl lg:text-2xl">
+              {showTicket ? userName : "Someone"} already claimed their ticket{" "}
+              <br />
+              for the Alveus Fall Carnival 2023:
+            </p>
+
+            <Wiggle
+              className="relative mx-auto my-10"
+              width={ticketConfig.width}
+              height={ticketConfig.height}
+              maskImage={ticketConfig.maskImage}
+            >
+              <div className="absolute inset-0 flex items-center justify-center bg-[#4E1362] text-2xl font-bold text-white">
+                {!showTicket && <span>Loading ticket…</span>}
+              </div>
+
+              {/* eslint-disable-next-line @next/next/no-img-element */}
+              <img
+                className={`relative w-full opacity-0 transition-opacity ${
+                  showTicket ? "block opacity-100" : "hidden"
+                }`}
+                src={getVirtualTicketImageUrl(eventId, userName)}
+                alt=""
+              />
+            </Wiggle>
+
+            <p className="text-center">
+              Sticker art by Mik_MWP (Sub emotes) <br />
+              and{" "}
+              <Link external href="https://www.instagram.com/lazygoosepixels/">
+                LazyGoose
+              </Link>{" "}
+              (Pixel Ambassadors)
+            </p>
```

Comment:
Can this be generated from the sticker pack config, instead of being hard-coded here?